### PR TITLE
ci: use git cliff on subcomponents

### PR DESCRIPTION
**Pull request by oldgiova**
Edit the Changelog and the body PR for the bundle version, plus the changelog for each component.

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -101,7 +101,7 @@ changelog:
     - test $GIT_CLIFF == "false" && echo "INFO - Skipping git-cliff" && exit 0
     - git remote add github-${CI_JOB_ID} https://${GITHUB_USER_NAME}:${GITHUB_BOT_TOKEN_REPO_FULL}@github.com/${GITHUB_REPO_URL} || true # Ignore already existing remote
     - gh repo set-default https://${GITHUB_USER}:${GITHUB_BOT_TOKEN_REPO_FULL}@github.com/${GITHUB_REPO_URL}
-    - RELEASE_PLEASE_PR=$(gh pr list --author "${GITHUB_USER_NAME}" --head "release-please--branches--${CI_COMMIT_REF_NAME}" --json number | jq -r '.[0].number // empty')
+    - RELEASE_PLEASE_PR=$(gh pr list --author "${GITHUB_USER_NAME}" --head "release-please--branches--${CI_COMMIT_REF_NAME}" | grep -v component | awk '{print $1}')
```
Comment (mzedel) [inline .gitlab-ci.yml:104]: not sure if this works to get the PR name:
```suggestion
    - RELEASE_PLEASE_BRANCH=$(gh pr list --author "${GITHUB_USER_NAME}" --head "release-please--branches--${CI_COMMIT_REF_NAME}" | grep -v component | rev | cut -d- -f1 | rev)
    - RELEASE_PLEASE_PR="release-please--branches--main--components--${RELEASE_PLEASE_BRANCH}"
```

**Review by mzedel — COMMENTED** [review]
In this case the bundle is for the most part not really interesting, but the components are the main focus 😕

**Comment by oldgiova** [comment]
> In this case the bundle is for the most part not really interesting, but the components are the main focus 😕

Iteration number 2: adding the changelog for every component

**Review by mzedel — APPROVED** [review]
Looks excellent to me 😃

**Pull request merged**